python/tests/transformers/tf_image_test.py

```diff
@@ -43,14 +44,42 @@ def _loadImageViaKeras(self, raw_uri):
         image = np.expand_dims(image, axis=0)
         return preprocess_input(image)
 
+    # TODO: I believe this is already tested in named_image_test, should we remove it?
     def test_load_image_vs_keras(self):
         g = tf.Graph()
         with g.as_default():
             image_arr = utils.imageInputPlaceholder()
+            # keras expects array in RGB order, we get it from image schema in BGR => need to flip
+            preprocessed = preprocess_input(imageIO._reverseChannels(image_arr))
+
+        output_col = "transformed_image"
+        transformer = TFImageTransformer(channelOrder='BGR',inputCol="image", outputCol=output_col, graph=g,
+                                         inputTensor=image_arr, outputTensor=preprocessed.name,
+                                         outputMode="vector")
+
+        image_df = image_utils.getSampleImageDF()
+        df = transformer.transform(image_df.limit(5))
+
+        for row in df.collect():
+            processed = np.array(row[output_col]).astype(np.float32)
+            # compare to keras loading
+            images = self._loadImageViaKeras(row["image"]['origin'])
+            image = images[0]
+            image.shape = (1, image.shape[0] * image.shape[1] * image.shape[2])
+            keras_processed = image[0]
+            self.assertTrue( (processed == keras_processed).all() )
+
+
+            # TODO: I believe this is already tested in named_image_test, should we remove it?
```

Comment:
let's keep this test and the one above as simple test cases for TFTransformer, one for BGR and one for RGB. what do you think?